<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_paramconnexion</class>
 <widget class="QDialog" name="dlg_paramconnexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paramètrage de connexion à la base</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="MDPlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLabel" name="Loginlabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>21</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MDPlineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>46</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginlineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>16</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="HelpupPushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Images/images.qrc">
      <normaloff>:/help.png</normaloff>:/help.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="AccesgroupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="title">
     <string>Emplacement du serveur</string>
    </property>
    <widget class="QRadioButton" name="PosteradioButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>Sur ce poste</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="LocalradioButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>55</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>Réseau local</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DistantradioButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>80</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>Accès distant (internet)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="IPlineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="IPlabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>IP ou DNS</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Portlabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>15</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PortcomboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="UpPushButton" name="AnnuluppushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="Images/images.qrc">
         <normaloff>:/Supprime.png</normaloff>:/Supprime.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>11</width>
         <height>11</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="UpPushButton" name="TestuppushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string>Tester</string>
       </property>
       <property name="icon">
        <iconset resource="Images/images.qrc">
         <normaloff>:/Test.ico</normaloff>:/Test.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="UpPushButton" name="OKuppushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="Images/images.qrc">
         <normaloff>:/start.png</normaloff>:/start.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpPushButton</class>
   <extends>QPushButton</extends>
   <header>uppushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LoginlineEdit</tabstop>
  <tabstop>MDPlineEdit</tabstop>
  <tabstop>AccesgroupBox</tabstop>
  <tabstop>PosteradioButton</tabstop>
  <tabstop>LocalradioButton</tabstop>
  <tabstop>DistantradioButton</tabstop>
  <tabstop>IPlineEdit</tabstop>
  <tabstop>PortcomboBox</tabstop>
  <tabstop>AnnuluppushButton</tabstop>
  <tabstop>TestuppushButton</tabstop>
  <tabstop>OKuppushButton</tabstop>
 </tabstops>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
